test/srt/cpu/test_topk.py

```diff
@@ -0,0 +1,173 @@
+import unittest
+
+import torch
+from sgl_kernel.common_ops import biased_grouped_topk_cpu as biased_grouped_topk
+from sgl_kernel.common_ops import grouped_topk_cpu as grouped_topk
+
+from sglang.test.test_utils import CustomTestCase
+
+
+# This is used by the Deepseek-V2 model
+class TestGroupedTopK(CustomTestCase):
```

Comment:
can we import that native impl from https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/layers/moe/topk.py